docs/plugins/lifecycle.md

```diff
@@ -17,9 +20,10 @@ The following states are available:
 | State         | Description                                 |
 | ------------- |:-------------------------------------------:|
 | init          | Application is initialising                 |
+| ready         | Application is ready for user interaction   |
```

Comment:
I understand the transition should be instant or quick, making the ready state a (very) short lived state. So from an App developer perspective it's probably not the most useful state to listen to.

Is there anything we can think of that would vouch against explicitly passing through the `ready` state?

Reply:
Hmm..

**Cons** of having `ready` as an explicit state:

1. keeping the number of states as few as possible
2. the app itself initiates this state, so it already knows
3. Anything it might want to do internally on `ready` should have been done before `ready()` was called
4. developers might wrongly assume that either `active` or `background` is specifically next and do the wrong thing during the `ready` listener.

Reply:
fair enough .. then let's remove the ready state all together, and just keep the `ready()` method as a one-way street to inform the platform

Reply:
👍